recipes/vape4d/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "vape4d" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/vape4d-{{ version }}.tar.gz
+  sha256: 6fbc1af308b317ef0cb59ba380088494ed8f05c3b3f38d079afece8329259ce3
+
+build:
+  script:
+    - export CARGO_PROFILE_RELEASE_STRIP=symbols
+    - export CARGO_PROFILE_RELEASE_LTO=fat
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+  number: 0
+  entry_points:
+    - vape4d = vape4d.vape4d:standalone
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('rust') }}
+    - cargo-make
+    - cargo-bundle-licenses
+    - maturin
+  host:
+    - python
+    - maturin >=1.0,<2.0
+    - pip
+  run:
+    - python
+    - numpy
+    - matplotlib-base
+    - seaborn
+    - cmasher
+    - cffi    
+    - libgcc
+
+test:
+  imports:
+    - vape4d
+  commands:
+    - pip check
+  requires:
+    - python
+    - pip
+
+about:
+  home: https://github.com/KeKsBoTer/vape4d
+  summary: Volume 4D Visualization
+  license: BSD-2-Clause
+  license_file: LICENSE
```

```suggestion
  license_file: 
    - LICENSE
    - THIRDPARTY.yml
```
Sorry I forgot we have to add the third-party licenses here.  This should be last change needed!